tests/README.md

```diff
@@ -47,6 +48,12 @@ Running specific storage tests modules:
 
     tox -e storage -- storage/image_test.py
 
+If you run only some tests then the configured coverage goals (see `tox.ini`)
+can not be met, so it's better to disable the coverage check and avoid the
+useless warning. Any arguments after `--` are passed through to pytest:
```

Comment:
Explaining why we need to use "--" should be done in the first example using it, maybe as an example.

We should really add --no-cov to all example running tests from single module, not as separate section. We can explain why on the first example using this.

Reply:
--no-cov does not collect coverage, which is bad. We always want to collect coverage so it is easy to inspect the coverage.

What we want to disable is failing the tests if coverage is lower than the limit, since this work only when running all the tests as root.

So I think this should be fixed by using:

   --cov-fail-under=0

as default value, so developers do not need to specify it, and enable the actual limit in the CI, when we run all the tests.

This can be done by using default in tox.ini:
https://github.com/oVirt/vdsm/blob/9bf9a30230ae318a7798a25b5ab362ce33a006a7/tox.ini#L97

We change the default to 0, and inject the value using
https://github.com/oVirt/vdsm/blob/master/automation/tests.sh

This is not in the scope of this PR, but can be the next step if you have time for this.

Reply:
Makes sense. I agree it's getting out of scope though.